Makefile

```diff
@@ -150,48 +155,62 @@ get-test-namespace: out/test-namespace
 .PHONY: e2e-setup
 e2e-setup: e2e-cleanup
 	$(Q)kubectl create namespace $(TEST_NAMESPACE)
+	$(Q)kubectl --namespace $(TEST_NAMESPACE) apply -f ./test/third-party-crds/postgresql_v1alpha1_database_crd.yaml
 
 .PHONY: e2e-cleanup
 e2e-cleanup: get-test-namespace
-	$(Q)-kubectl delete namespace $(TEST_NAMESPACE) --timeout=10s --wait
+	$(Q)kubectl delete namespace $(TEST_NAMESPACE) --timeout=45s --wait || true
```

Comment:
Why did you remove `-` and used `|| true`?

Reply:
Okay, I just learnt something about Make targets. Fixing this.